lib/trixer/slotter.rb

```diff
@@ -192,11 +192,16 @@ def place_slot_data(place_id:)
           current_booking_length += 1
           current_length = 1
           slot_data[:free_duration] = { rest: 0, booking.id => current_booking_length - 1 }
+          current_booking = nil if @gap_slots[slot] == true
         elsif @gap_slots[slot] == true
           leng = current_length
           current_length = 1
           current_booking_length = 1
+          current_booking = nil
           slot_data[:free_duration] = { rest: leng }
+        elsif !booking && current_booking
+          current_length += 1
```

Comment:
I think you need to increment `current_booking_length` here as well

Reply:
see comment above